nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/memory/deallocation/DeallocatorService.java

```diff
@@ -37,39 +35,6 @@
 import java.util.concurrent.ConcurrentSkipListMap;
 import java.util.concurrent.atomic.AtomicLong;
 
-/**
- * Handles deallocation of offheap references.
- * This service starts a set of gc threads
- * and reference queues.
- * These reference queues are passed references to the jvm.
- *
- * The instances passed are of {@link Deallocatable}
- * These are all pointers with a deallocate() method which will be
- * called before being finally garbage collected by the JVM.
- *
- * The deallocator service handles generating unique ids by using an
- * atomic counter + an offset generated by {@link Deallocatable#getUniqueId()}
- *
- * Note that some buffers also may not be deallocated. Before deallocation a buffer
- * is checked for whether it's constant with {@link Deallocator#isConstant()}
- * If a reference is constant (this is usually only applicable to shape buffers)
- * then it will not be deallocated.
- *
- * You may turn the deallocator service off by setting {@link #noPointerGc}
- * to true with the system property: {@link ND4JSystemProperties#NO_ARRAY_GC}
- *
- * Note the service also checks "org.bytedeco.javacpp.noPointerGC"
- *
- * Also of note is you may access the references with {@link #getReferenceMap()}
- * DO THIS AT YOUR OWN RISK. ONLY FOR advanced users.
- *
- *
- * Known limitations:
- *
- * Due to relying on the JVM to supply garbage collectable references,
- * the allocator fall behind in deallocation. This happens VERY rarely and only at certain scales.
- *
- */
```

Comment:
Why do you remove that comment again?

Reply:
Mixed around in different commits. Thanks for highlighting. Added back.